Source/UI/PaymentForm/ViewModel/PaymentViewModel.swift

```diff
@@ -23,6 +23,7 @@ protocol PaymentViewModel {
   var cardTokenRequested: ((Result<TokenDetails, TokenisationError.TokenRequest>) -> Void)? { get set }
   func updateAll()
   func viewControllerWillAppear()
+  func viewControllerCancelled()
```

Comment:
The name of the method viewControllerCancelled and what it does not match. Should we rename this to `logPaymentFormDismissed` ?

Reply:
I wanted to name dismiss, but looking at the code, the screen is also dismissed on completion of the journey (tokenisation completed). That's what was originally planned but can be misguiding when seen out of codebase?
Next best thing was cancelled, as the event gets triggered on user pressing "back arrow" from navigation bar, which is the closest thing to a cancel that the journey offers.